src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link should point to a CSS file, not a SCSS file. SCSS is a preprocessor language and needs to be compiled to CSS before it can be used in an HTML file. Please ensure the href attribute points to a valid CSS file.